tests/test_shell_bash_background_helpers_disowned.py

```
@@ -0,0 +1,217 @@
+                    break
+                output.extend(chunk)
+
+            waited_pid, status = os.waitpid(pid, os.WNOHANG)
+            if waited_pid == pid:
+                exit_status = os.waitstatus_to_exitcode(status)
+                break
+        else:
+            try:
+                os.kill(pid, 15)
+            except ProcessLookupError:
+                pass
```

Ruff (0.15.13)

[warning] Use `contextlib.suppress(ProcessLookupError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(ProcessLookupError): ...`

(SIM105)